.github/workflows/pipeline-infrastructure-cleardown.yaml

```diff
@@ -46,11 +31,10 @@ jobs:
       domain: "uec-dos-management"
       stacks:
         "[
+          'lambda-stack',
           'test-stack-1',
```

Comment:
Are we still deploying test-stack-1 , test-stack-2 etc. Are we clearing down only what we need to ?